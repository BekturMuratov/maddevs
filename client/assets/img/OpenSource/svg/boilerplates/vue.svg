<svg xmlns="http://www.w3.org/2000/svg" width="336" height="125" fill="none" xmlns:v="https://vecta.io/nano"><g clip-path="url(#A)"><path d="M206.784.001l-38.797 67.203L129.191.001H-.008l167.996 290.997L335.984.001h-129.2z" fill="#41b883"/><path d="M206.785.001l-38.797 67.203L129.191.001h-62l100.798 174.595L268.786.001h-62.001z" fill="#34495e"/></g><defs><clipPath id="A"><path fill="#fff" d="M0 0h336v291H0z"/></clipPath></defs></svg>