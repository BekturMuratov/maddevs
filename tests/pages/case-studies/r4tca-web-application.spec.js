import { render } from '@testing-library/vue'
import InternationalTradeCenter from '@/pages/case-studies/R4TCA-web-application'

const mocks = {
  $getMediaFromS3: () => 'img.jpg',
  $lazyLoad: {
    init: jest.fn(),
  },
}

const stubs = ['NuxtLink', 'Main']

describe('InternationalTradeCenter root component', () => {
  it('should render correctly', () => {
    const { container } = render(InternationalTradeCenter, {
      mocks,
      stubs,
    })

    expect(container).toMatchSnapshot()
  })
})
