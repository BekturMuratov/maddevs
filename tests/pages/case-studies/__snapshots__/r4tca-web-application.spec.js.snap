// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InternationalTradeCenter root component should render correctly 1`] = `
<div>
  <main
    class="case case--veeqo"
  >
    <section
      class="case_header case_header-itc"
      id="transparent-header-area"
    >
      <div
        class="case_main-video_fallback"
        style="background-image: url(img.jpg);"
      />
       
      <video
        autoplay="autoplay"
        class="case_main-video media_lazy"
        data-testid="test-case_main-video"
        loop="loop"
        muted="muted"
      >
        <source
          data-src="img.jpg"
          type="video/mp4"
        />
        
    Your browser does not support the video tag.
  
      </video>
       
      <div
        class="case_header-content"
      >
        <div
          class="case_header-text"
          style="opacity: 1;"
        >
          <div
            class="case_case-study-item"
          >
            
        Case Study
      
          </div>
           
          <h1
            class="case_header-title"
          >
            
      Ready4Trade 
            <br />
             Central Asia
    
          </h1>
           
          <p
            class="case_header-description"
          >
            
      A web app customized for the delivery of export management coaching 
            <br />
             missions that contributes to the development of 
            <span
              class="case_nowrap"
            >
              intra-regional
            </span>
             
            <br />
             and international trade in Central Asia
    
          </p>
           
        </div>
         
        <img
          alt="International trade center logo"
          class="case_header-logo case_itc-logo"
          height="154.95"
          src="img.jpg"
          style="opacity: 1;"
          width="377"
        />
      </div>
    </section>
     
    <main-stub />
     
    <section
      class="case_footer container_full"
    >
      <img
        alt="Optimization for Veeqo"
        class="case_logotype-veeqo img_lazy"
        data-src="[object Object]"
        height="72"
        width="195"
      />
       
      <h2
        class="case_title case_title_h2 case_title-veeqo"
      >
        
    Optimization for Veeqo
  
      </h2>
       
      <nuxtlink-stub
        class="case_next-project-button"
        to="/case-studies/veeqo/"
      >
        
    Read next project
  
      </nuxtlink-stub>
    </section>
  </main>
</div>
`;
